.vscode/settings.json

```diff
@@ -0,0 +1,116 @@
+{
+	// VSCode Custom Settings
+	"typescript.tsdk": "node_modules/typescript/lib",
+	// ESLint
+	"eslint.format.enable": true,
+	"eslint.validate": [
+		"javascript",
+		"typescript",
+		"javascriptreact",
+		"typescriptreact"
+	],
+	"[javascript]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[typescript]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[javascriptreact]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[typescriptreact]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	// PHP Code Sniffer
+	"phpCodeSniffer.autoExecutable": true,
+	"phpCodeSniffer.standard": "Custom",
+	"phpCodeSniffer.standardCustom": "WordPress-VIP-Go,WordPress-Docs",
+	// PHP Code Beautifier
+	"phpcbf.standard": "WordPress-VIP-Go,WordPress-Docs",
+	"[php]": {
+		"editor.defaultFormatter": "obliviousharmony.vscode-php-codesniffer"
+	},
+	// PHPStan
+	"phpstan.timeout": 30000,
```

Comment:
I have tested the settings on my other Macbook and due to timeout phpstan was not working so I increased the timeout from default 10s to 30s just to make sure that it works for most devs without any configs.